server/sonar-server/src/main/java/org/sonar/server/user/ws/CurrentAction.java

```diff
@@ -138,7 +141,7 @@ private CurrentWsResponse toWsResponse(DbSession dbSession, String userLogin) {
 
   private void setHomepageParameter(DbSession dbSession, String homepageType, @Nullable String homepageParameter, CurrentWsResponse.Homepage.Builder homepage) {
```

Comment:
suggestion: to have a validateRequest method that checks that organization is provided when it's required etc. with a proper message each time